bundles/core/org.eclipse.smarthome.core.thing.xml.test/src/test/resources/test-bundle-pool/SystemChannels.bundle/ESH-INF/i18n/SystemChannels_de.properties

```diff
@@ -1,5 +1,7 @@
 channel-type.system.my-channel.label = Mein String My Channel
 channel-type.system.my-channel.description = Wetterinformation mit My Channel Type Beschreibung
 
+channel-type.system.low-battery.label = Niedriger Batteriestatus
```

Comment:
I am not sure if this is what we want to do here. When defining a things channel using a default system channel type we expect that the label and description are inherited and so the translations for them.

Does it maybe makes more sense to switch from `ChannelTypeRegistry` to `ChannelTypeProvider` here as well? wdyt?

Reply:
Good point, thanks. I have changed it.